commands/install_license

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [--license <license-key>]
+Ex: c8y oee \install_license --license AbcD4E567
+Install Apamma license via c8y tenant option
+EOF
+}
+
+# Parse command-line options
+REST_ARGS=()
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -h|--help)
+      help
+      exit 0
+      ;;
+    -l|-license|--license)
```

Comment:
is autocompletion supported for such parameters?

Reply:
As far as I know, autocomplete feature implemented by Reuben only supports for API yaml definition.

Reply:
I will do it after finishing this PR so that Reuben can have idea what he should do